src/main/java/de/tum/in/www1/artemis/service/iris/settings/IrisSettingsService.java

```diff
@@ -144,6 +149,15 @@ private void initializeIrisChatSettings(IrisGlobalSettings settings) {
         settings.setIrisChatSettings(irisChatSettings);
     }
 
+    private void initializeIrisLectureIngestionSettings(IrisGlobalSettings settings) {
+        var irisLectureIngestionSettings = settings.getIrisLectureIngestionSettings();
+        if (irisLectureIngestionSettings == null) {
+            irisLectureIngestionSettings = new IrisLectureIngestionSubSettings();
+            irisLectureIngestionSettings.setEnabled(false);
+        }
```

Comment:
This behavior is now duplicated multiple times. Can you move this into a common method?